cli/maintainer/src/commands_multisig.rs

```diff
@@ -448,6 +461,46 @@ enum SolidoInstruction {
     },
 }
 
+#[allow(clippy::enum_variant_names)]
+#[derive(Serialize)]
+enum AnkerInstruction {
+    ChangeTerraRewardsDestination {
+        #[serde(serialize_with = "serialize_b58")]
+        anker_instance: Pubkey,
+
+        #[serde(serialize_with = "serialize_b58")]
+        manager: Pubkey,
+
+        old_terra_rewards_destination: TerraAddress,
+
+        new_terra_rewards_destination: TerraAddress,
+    },
+    ChangeTokenSwapPool {
+        #[serde(serialize_with = "serialize_b58")]
+        anker_instance: Pubkey,
+
+        #[serde(serialize_with = "serialize_b58")]
+        manager: Pubkey,
+
+        #[serde(serialize_with = "serialize_b58")]
+        old_token_swap_pool: Pubkey,
+
+        #[serde(serialize_with = "serialize_b58")]
+        new_token_swap_pool: Pubkey,
+    },
+    ChangeSellRewardsMinBps {
+        #[serde(serialize_with = "serialize_b58")]
+        anker_instance: Pubkey,
+
+        #[serde(serialize_with = "serialize_b58")]
+        manager: Pubkey,
+
+        old_sell_rewards_min_bps: u64,
```

Comment:
```suggestion
        old_sell_rewards_min_out_bps: u64,
```
Nit: for consistency with the field in the `Anker` struct. Also below.